### DISCUSSION TITLE
Why did you choose Apache-2.0 instead of MIT as license ?

### DISCUSSION THREAD

**Discussion by johnsonthedev** [joe-bell/cva#25] [Q&A] [answered]
Hi, 

Thanks for the lib. I found it while searching for a lib similar to stitches syntax to build component variants with tailwind. 
From your description it seems exactly what I was looking for except the license. Do you mind explaining why you choosed Apache instead of MIT for the license ?

Thanks ✌️

**Comment by joe-bell** [comment, marked as answer]
I've poured a lot of time any energy into my open-source work, Apache is the only LICENSE – in my view – that respects this by encouraging proper credit
